html/doc/system.html

```diff
@@ -475,6 +475,18 @@ <h4 id="redis">Configuring Redis</h4>
   <dt>Nginx:<dd><pre class="prettyprint"
      >pagespeed RedisReconnectionDelayMs timeout_in_milliseconds;</pre>
 </dl>
+    <p>
+      You can also select the Redis logical database having the specified numeric index.
+      This setting defaults to database index 0.
+      Note that the Redis clustering feature does not allow specifying a database index,
+      and therefore is incompatible with this setting:
+    </p>
+<dl>
+  <dt>Apache:<dd><pre class="prettyprint"
+     >ModPagespeedRedisDatabaseSelectIndex index</pre>
```

Comment:
Is this a DatabaseSelectIndex, or are we selecting a DatabaseIndex?  Just want to make sure we get a name here that makes sense to Redis people.  If it's really called a DatabaseSelectIndex then this is the right spelling, and we should also change the param below from "int database_index" to "int database_select_index". 

  If we are selecting a DatabaseIndex then we should remove the "Select" from the name.

Reply:
We are selecing database index. So modifying property name to ModPagespeedRedisDatabaseIndex